source/release-notes/1.3.txt

```diff
@@ -22,4 +22,11 @@ Capped Collections
 
 Starting in 1.3.0, {+c2c-product-name+} supports :ref:`capped
 collections <manual-capped-collection>`. For more information, see
-:ref:`c2c-capped-collections`.
\ No newline at end of file
+:ref:`c2c-capped-collections`.
+
+Disable Index Builds
```

Comment:
Can you add non-essential or non-required here. Don't want to give the impression its for every index.

Reply:
Fixed